src/server/services/pullRequest.js

```diff
@@ -24,7 +24,7 @@ var commentText = function (signed, badgeUrl, claUrl, user_map) {
 			text += '<br/>:x: ' + signee;
 		});
 	} else {
-		text = '[![CLA assistant check](' + badgeUrl + ')](' + claUrl + ') <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>';
+		text = '[![CLA assistant check](' + badgeUrl + ')](' + claUrl + ') <br/>Thank you for your submission, we really appreciate your work. Like many open source projects, before we can accept your contribution, we first ask that you sign our [Contributor License Agreement](' + claUrl + ').<br/>';
```

Comment:
As someone who has a knee-jerk negative reaction to EULAs and the like, I
think a gentle reminder that however distasteful this is standard practice
would at least somewhat mollify me.

I'm certainly open to other phrasings, maybe more matter-of-fact like "As
an open source project, ..." ('cos, it is/should be standard practice,
right)?

Han

On Mon, Mar 14, 2016 at 1:35 AM, Anton <notifications@github.com> wrote:

> In src/server/services/pullRequest.js
> <https://github.com/cla-assistant/cla-assistant/pull/110#discussion_r55967894>
> :
>
> > @@ -24,7 +24,7 @@ var commentText = function (signed, badgeUrl, claUrl, user_map) {
> >  			text += '<br/>:x: ' + signee;
> >  		});
> >  	} else {
> > -		text = '[![CLA assistant check](' + badgeUrl + ')](' + claUrl + ') <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>';
> > +		text = '[![CLA assistant check](' + badgeUrl + ')](' + claUrl + ') <br/>Thank you for your submission, we really appreciate your work. Like many open source projects, before we can accept your contribution, we first ask that you sign our [Contributor License Agreement](' + claUrl + ').<br/>';
>
> It looks much better now.
> I would only skip the "Like many open source projects" part
> @dfarr <https://github.com/dfarr> , @thojansen
> <https://github.com/thojansen> : what do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cla-assistant/cla-assistant/pull/110/files#r55967894>.
>